node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/HostResumeProvisioner.java

```diff
@@ -65,7 +67,7 @@ protected double maintain() {
                                       " children, failing out the host recursively", e);
                 // Fail out as operator to force a quick redeployment
                 nodeRepository().nodes().failOrMarkRecursively(
-                        host.hostname(), Agent.DynamicProvisioningMaintainer, "Failed by HostProvisioner due to provisioning failure");
+                        host.hostname(), Agent.operator, "Failed by HostProvisioner due to provisioning failure");
```

Comment:
This was `operator` originally (see comment above), changed to DPM in #23345

Reply:
hm, could quick redeployment be triggered for a HostResumeProvisioner agent too?

Reply:
Done